components/organisms/Header.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types";
+import Image from "next/image";
+import { SearchBar } from "../atoms/SearchBar";
+import { useState } from "react";
+
+export function Header(props) {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
+
+  return (
+    <div className="w-full  mt-6 ">
+      <div className="w-full h-auto flex  md:flex-row justify-center  mt-4">
+        <div className="w-full flex  flex-col  lg:flex-col  xl:flex-row   justify-between relative">
+          <div className="md:h-9 h-6 mb-4">
+            <Image
+              className="h-full  bottom-0 left-0"
+              width={300}
+              height={80}
+              src={props.headerLogoImage}
+              alt={props.headerLogoAltText}
+            />
+          </div>
```

Comment:
The header looks really good, actually.

the one thing which I missed in my earlier review is that on small sizes, there is a bit too much padding vs canada.ca. The fix here is to adjust the "layout-container" class padding/margin for smaller screen sizes.\

It's a bit hard to tell, but there is more side padding in our site (on the left side).
<img width="1439" alt="Screen Shot 2021-04-09 at 11 11 57" src="https://user-images.githubusercontent.com/2454380/114201872-aa4d5800-9924-11eb-8f8b-962c2b1abea5.png">